src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/DerivativeEffect.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Text.RegularExpressions.Symbolic
+{
+    /// <summary>
+    /// Describes effects to be applied to registers.
```

Comment:
Can you expand some of these descriptions, maybe in a `<remarks/>` section?  What are "effects" in this context?  What are "registers" in this context?  Etc.